src/OpenRasta/Diagnostics/TraceSourceLogger.cs

```diff
@@ -4,6 +4,11 @@
 
 namespace OpenRasta.Diagnostics
 {
+  public static class DefaultLogger
+  {
+    public static ILogger Instance { get; set; } = new TraceSourceLogger();

```

Comment:
I think there's a `null` logger. Could we default to that?

Reply:
I left it as `TraceSourceLogger` to be less break-y, but we could certainly use `NullLogger.Instance` here instead.

Reply:
Suggest we leave it as-is to make sure we don't hide any Warnings/Errors during startup. By the time per-request tracing might happen we'll have had a chance to replace this: https://github.com/ololabs/openrasta-core/pull/6#discussion_r1825907857.